public/robots.txt

```diff
@@ -0,0 +1,11 @@
+# *
+User-agent: *
+Allow: /
+
+# Host
+Host: https://developer.hashicorp.com
+
+# Sitemaps
+Sitemap: https://developer.hashicorp.com/sitemap.xml
+Sitemap: https://developer.hashicorp.com/hcp-sitemap.xml
+Sitemap: https://developer.hashicorp.com/sitemap-docs.xml
```

Comment:
This is generated by `next-sitemap` too, right? Should we `.gitignore` this file?

Reply:
Ignored all the files generated by sitemap stuff 😄